oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreService.java

```diff
@@ -671,6 +680,10 @@ protected void deactivate() {
             docStoreEmbeddedVerificationFeature.close();
         }
 
+        if (prevNoPropCacheFeature != null) {
```

Comment:
Though this is unrelated to this PR, could you please add a utility method that accepts an array of `Feature` and closes them all?

Reply:
I'd suggest to move this to a separate ticket as it is unrelated. +1 for changing it though (but it could happen with lower prio). Wdyt?

Reply:
jira OAK-11248 created

Reply:
thx!